### PR TITLE
tflint: Add version 0.13.1

### PR DESCRIPTION
**Pull request by stevehipwell**
Add tflint package

### CODE REVIEW

File: bucket/tflint.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "url": "https://github.com/terraform-linters/tflint/releases/download/v0.13.1/tflint_windows_amd64.zip",
+            "hash": "715ff0539130d2a5c4b35f8021fe7204108c8487442a4f28857d9f5bdfee6799"
+        },
+        "32bit": {
+            "url": "https://github.com/terraform-linters/tflint/releases/download/v0.13.1/tflint_windows_386.zip",
+            "hash": "d7eca9d0843ff02a38e8981c82bf0f8caec560f9878650f4b7407e8da1e588cf"
+        }
+    },
+    "bin": "tflint.exe",
+    "depends": "terraform",
```
Comment (Ash258) [inline bucket/tflint.json]:
```suggestion
```

File: bucket/tflint.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/terraform-linters/tflint",
+    "description": "A Terraform linter focused on possible errors, best practices, etc.",
+    "license": "Mozilla Public License 2.0",
+    "version": "0.13.1",
+    "architecture": {
```
Comment (Ash258) [inline bucket/tflint.json]:
```suggestion
    "depends": "terraform",
    "architecture": {
```

Comment (Ash258) [reply bucket/tflint.json]: Why is this depend needed in first time?

I can check simple alfa.tf with reference to variables.tf without any issue.

image


Only issue is when I change POLICY to POLICYa

image

File: bucket/tflint.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "https://github.com/terraform-linters/tflint",
+    "description": "A Terraform linter focused on possible errors, best practices, etc.",
+    "license": "Mozilla Public License 2.0",
```
Comment (Ash258) [inline bucket/tflint.json]:
```suggestion
    "license": "MPL-2.0",
```

File: bucket/tflint.json (hunk 2 above)
Comment (stevehipwell) [reply bucket/tflint.json]: @Ash258 I was working from the docs so I missed this. It should be removed by your first requested change.

**Pull request merged**